addons/open-in-gandi/addon.json

```diff
@@ -0,0 +1,20 @@
+{
+    "versionAdded": "1.0.0"
```

Comment:
You're missing a comma.

Reply:
ohhh(this guy forgot to add a comma, laugh at him)